.github/workflows/testing.yml

```
@@ -0,0 +1,253 @@
+
+      - name: Create issue for persistent failures
+        if: failure()
+        uses: actions/github-script@ed597411d8f924073f98dfc5c65a23a2325f34cd # v8
+        with:
+          script: |
+            github.rest.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: '🔧 Self-healing failed - Manual intervention required',
+              body: `## Failed Tests Summary
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)